src/material/slider/slider.html

```diff
@@ -7,6 +7,16 @@
   <div class="mdc-slider__track--active">
     <div #trackActive class="mdc-slider__track--active_fill"></div>
   </div>
+  <div class="mdc-slider__end-tick-mark-container">
+    @if (_cachedWidth) {
```

Comment:
Why can't we reuse the rendering of the tick marks below?

Reply:
The rendering of the tickMarkContainer div and tick marks below is based on the showTickMarks flag. Since I want this end tick mark to be on the slider permanently regardless of the value of the showTickMarks flag, I thought to make a permanent div to hold the end tick mark. 

Let me know if you recommend approaching it a different way!

Reply:
If it's always supposed to be there, couldn't we hardcode one `div` and position it with CSS?